src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -963,4 +963,21 @@ namespace OpenLoco
             GameCommands::doCommand(args, GameCommands::Flags::apply);
         }
     }
+
+    void removeEntityFromThought(AiThought& thought, EntityId id)
+    {
+        auto iter = std::find(std::begin(thought.var_66), std::end(thought.var_66), id);
+        if (iter == std::end(thought.var_66))
+        {
+            return;
+        }
+        // Original would copy the value from var_66 + 2 which
+        // would mean it would copy currency var_76 if var_44 was 7
+        // I don't think that is possible but lets just add an assert.
+        assert(thought.var_44 < 7);
```

Comment:
similarly can we name `var_44`?

Reply:
I've looked at var_44 and var_66 again and I still don't know enough to name them. I think best to leave it for now.